### PR TITLE
Do not truncate articles in RSS feed

### PR DESCRIPTION
**Pull request by fcrozat**
This allows to read blog articles in RSS feed reader directly.

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -18,7 +18,7 @@ layout: null
           <link href="{{ site.url }}{{ post.url }}/"/>
           <updated>{{ post.date | date_to_xmlschema }}</updated>
           <id>{{ site.url }}{{ post.url }}/</id>
-          <summary type="html">{{ post.content | strip_html | truncatewords: 75 }}</summary>
+          <summary type="html">{{ post.content | strip_html }}</summary>
```
Comment (hennevogel) [inline feed.rss]: Can you limit the amount of posts to 10 please (line 15)? Otherwise this might get very big...

Comment (fcrozat) [reply feed.rss]: done

File: feed.rss [hunk 2]
```diff
@@ -12,7 +12,7 @@ layout: null
     <author><name>OBS Team</name></author>
     <rights>Copyright (c) 2012 OBS Team</rights>
 
-    {% for post in site.posts limit:20 %}
+    {% for post in site.posts limit:10 }
       <entry>
```
Comment (eduardoj) [inline feed.rss]: Missing `%` before `}`

Comment (fcrozat) [reply feed.rss]: my bad, I read 20% not 20

**Comment by bgeuken** [comment]
Thanks again for your contribution:-)

**Pull request merged**